Fix:
tclexpr.c:1321:1: error: implicit declaration of function 'Tcl_ReturnDouble' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
Tcl_ReturnDouble (interp, value.doubleValue);
^
--- src/tclx/ucbsrc/tclexpr.sed.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/tclx/ucbsrc/tclexpr.sed	2022-05-16 20:43:13.000000000 -0500
@@ -1,2 +1,5 @@
+/#include "tclint.h"/a\
+\#include "tclxint.h"
+
 /	    sprintf(interp->result, "%g", value.doubleValue);/c\
 	    Tcl_ReturnDouble (interp, value.doubleValue);
